ui/js/controllers/bugfiler.js

```diff
@@ -31,13 +31,24 @@ treeherder.controller('BugFilerCtrl', [
             return reftest !== "";
         };
 
+        $scope.search_terms = search_terms;
         $scope.parsedLog = parsedLog;
         $scope.fullLog = fullLog;
         $scope.crashSignatures = crashSignatures.join("\n");
         if ($scope.isReftest()) {
             $scope.reftest = reftest;
         }
 
+        $scope.isSummaryBlacklisted = function() {
```

Comment:
The name of this function implies it returns a bool, when it returns a string.
Could you switch it to something else?